Zig.gitignore

```diff
@@ -0,0 +1,18 @@
+# This file is for zig-specific build artifacts.
+# If you have OS-specific or editor-specific files to ignore,
+# such as *.swp or .DS_Store, put those in your global
+# ~/.gitignore and put this in your ~/.gitconfig:
+#
+# [core]
+#     excludesfile = ~/.gitignore
+#
+# Cheers!
+# -andrewrk
```

Comment:
Not needed.

Reply:
https://github.com/el10savio/gitignore/commit/08f8a14aeefbc399c6a53878a5bf45ee9e3439c4